backend/src/main/java/fk/prof/backend/util/ZookeeperUtil.java

```diff
@@ -2,15 +2,27 @@
 
 import io.vertx.core.Future;
 import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.utils.ZKPaths;
+import org.apache.zookeeper.CreateMode;
 import org.apache.zookeeper.KeeperException;
 
+import java.util.List;
+
 public class ZookeeperUtil {
 
+  private static final String DELIMITER = "/";
+
   public static byte[] readZNode(CuratorFramework curatorClient, String zNodePath)
       throws Exception {
     return curatorClient.getData().forPath(zNodePath);
   }
 
+  public static byte[] readLatestSeqZNodeChild(CuratorFramework curatorClient, String zNodePath) throws Exception {
+    List<String> sequentialPolicies = ZKPaths.getSortedChildren(curatorClient.getZookeeperClient().getZooKeeper(), zNodePath);
+    zNodePath = zNodePath + DELIMITER + sequentialPolicies.get(sequentialPolicies.size() - 1);
```

Comment:
Will make it return null in that case. Earlier it would have thrown up a np exception.